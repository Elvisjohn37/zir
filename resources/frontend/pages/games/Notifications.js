import React from 'react';
import Container from '@mui/material/Container';
import OnPageError from 'frontend/components/OnPageError';

export function ErrorMessageDisplay({ className, response }) {
	return (
		<Container maxWidth="lg" className={className}>
			<OnPageError response={response} />
		</Container>
	);
}
