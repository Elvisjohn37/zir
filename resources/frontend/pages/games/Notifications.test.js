import React from 'react';
import { ErrorMessageDisplay } from './Notifications';
import { render } from '@testing-library/react';

jest.mock('@mui/material/Container');
jest.mock('frontend/components/OnPageError');

describe('<ErrorMessageDisplay />', () => {
	const SAMPLE_RESPONSE = { data: 'test data' };

	it('should match snapshot', () => {
		let { asFragment } = render(<ErrorMessageDisplay response={SAMPLE_RESPONSE} />);
		expect(asFragment()).toMatchSnapshot();
	});
});
