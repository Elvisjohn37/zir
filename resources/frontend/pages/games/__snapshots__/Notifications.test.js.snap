// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<ErrorMessageDisplay /> should match snapshot 1`] = `
<DocumentFragment>
  <div
    data-testid="Container"
    maxwidth="lg"
  >
    <div
      data-testid="default"
      response="{\\"data\\":\\"test data\\"}"
    />
  </div>
</DocumentFragment>
`;
